src/instructlab/sdg/generate_data.py

```diff
@@ -210,15 +210,28 @@ def _context_init(
         **extra_kwargs,
     )
 
-
 def _sdg_init(ctx, pipeline):
     pipeline_pkg = None
 
     # Search for the pipeline in User and Site data directories
     # then for a package defined pipeline
     # and finally pipelines referenced by absolute path
-    data_dirs = [os.path.join(xdg_data_home(), "instructlab", "sdg")]
-    data_dirs.extend(os.path.join(dir, "instructlab", "sdg") for dir in xdg_data_dirs())
+    data_dir = os.path.join(xdg_data_home(), "instructlab", "sdg", "models", "docling")
+    data_dirs = [data_dir]
+    data_dirs.extend(
+        os.path.join(dir, "instructlab", "sdg", "models", "docling") for dir in xdg_data_dirs()
+    )
```

Comment:
changing the base data dir to be `instructlab/sdg/models/docling` is probably not a good idea. you still want to look for all other assets under `instructlab/sdg` the way it currently works

I would just try and replicate the existing code flow for how we read other assets now and extend it to look for `models/docling-models`

Reply:
addressed